frame/bounties/src/lib.rs

```diff
@@ -185,6 +220,43 @@ pub enum BountyStatus<AccountId, BlockNumber> {
 	},
 }
 
+/// A Subbounty proposal.
+#[derive(Encode, Decode, Clone, PartialEq, Eq, RuntimeDebug)]
+pub struct SubBounty<AccountId, Balance, BlockNumber> {
+	/// The subcurator fee.
+	fee: Balance,
+	/// The deposit of subcurator.
+	curator_deposit: Balance,
+	/// The status of this subbounty.
+	status: SubBountyStatus<AccountId, BlockNumber>,
+}
+
+/// The status of a bounty proposal.
+#[derive(Encode, Decode, Clone, PartialEq, Eq, RuntimeDebug)]
+pub enum SubBountyStatus<AccountId, BlockNumber> {
+	/// The Subbounty is added and waiting for curator assignment.
+	Added,
+	/// A Subcurator has been proposed by the `curator`. Waiting for acceptance from the subcurator.
+	SubCuratorProposed {
+		/// The assigned subcurator of this bounty.
+		subcurator: AccountId,
+	},
+	/// The subbounty is active and waiting to be awarded.
+	Active {
+		/// The subcurator of this subbounty.
+		subcurator: AccountId,
```

Comment:
similar to above.

Reply:
Sure, refactor done :+1:

Reply:
Since its minor, the proposed change is breaking the test code. May be wish to take it as separate PR Kian.